pkg/ocm/ocmWrapper.go

```diff
@@ -45,9 +46,26 @@ type DefaultOCMInterfaceImpl struct{}
 
 var DefaultOCMInterface OCMInterface = &DefaultOCMInterfaceImpl{}
 
+func (o *DefaultOCMInterfaceImpl) SetupOCMConnection() (*ocmsdk.Connection, error) {
+	ocmNotLoggedInMessage := "Not logged in"
+
+	// Setup connection at the first try
+	connection, err := ocm.NewConnection().Build()
```

Comment:
pls hook `OCM_URL` env hack for fixing the reported issue
```
urlenv := os.Getenv("OCM_URL")
	if urlenv != "" {
		url, _ := urls.ResolveGatewayURL(urlenv, nil)
		os.Setenv("OCM_URL", url)
	}
```

Reply:
yeah, I was about to include this part into the code... not finished yet